app/components/bali/avatar/previews/with_image.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_with(url: '/') do |form| %>
+  <%= render(Bali::Avatar::Component.new(form: form, attr_model_name: :avatar, 
+                                         accepted_formats: '.jpg, .jpeg, .png')) do |c| %>
+
+    <% c.picture(picture: image_url) %>
```

Comment:
it's a bit repetitive to have to say c.picture(picture:) and the name of the parameter doesn't give you any indication of the type of argument it expects